static_src/util/cf_api.js

```diff
@@ -28,6 +51,36 @@ function handleError(err, errHandler = errorActions.errorFetch) {
   }
 }
 
+function parseError(resultOrError) {
```

Comment:
Yeah, I believe so because if the axios request is successful, we have a response with data from CF API (and maybe a non-200 status). But if there's a network issue, it would just be an error.